src/processing/1_Rulling av bilder/Rulling av bilder.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Rullering av bilder
+level: 1
+author: Kine Gjerstad Eide
+language: nb
+---
+
+# Introduksjon {.intro}
+
+I denne oppgaven skal vi se på hvordan vi kan ha to bilder ved siden av hverandre som beveger seg til siden, og når det ene bildet er på vei ut av vinduet, så hopper det til høyre siden av det andre bildet slik at det ser ut som om vi har et langt bilde som beveger seg.
+
+Når all koden er skrevet ferdig vil du ha begynnelsen av et spill som ser slik ut:
+
+![](preview.png)
+
+# Steg 1: Oppsett {.activity}
+Først må vi sette opp programmet vårt med en `draw `og en `setup` metode
+
+![](setup.png "draw og setup")
+
+nå kan vi lagre programmet vårt slik at vi kan finne det igjen senere. Trykk `ctrl`+ `s` på tastaturet og gi det et navn som du husker, for eksempel `rulling`.
+
+Nå kan du også høyretrykke på dette bildet av en bakke, og lagre det i samme mappe som spillet ditt ble lagret i. Processing lagrer filer i `.pde` format.
+
+![](bakke.png "bakkebit") 
+
+Så kan vi sette opp størrelsen på vinduet og en bakgrunnsfarge i vinduet.
+
+## Gjør dette {.check}
+- [ ] I `setup` metoden, altså mellom krøllparentesene etter `setup` skriv: `size(800, 800);`
+- [ ] I `draw` metoden, skriv: `background(100, 100, 200);`
+
+## Forklaring {.try}
+Både `size` og `background` er ferdigskrevne metoder som vi bare bruker. Å bruke ferdige metoder  kalles å gjøre metodekall.
+
+Tallene i metodekallet på `size` betyr hvor høy og hvor bredt vinduet skal være.
+Tallene i `background` bestemer hvor mye rødt, grønt og blått det skal være.
+Hvert av de tallene kan bare være mellom 0 og 255. 
+
+Alt som står inni parentesene på en metode vi bruker er argumentene, eller parameterne til metoden. 
+
+## Gjør dette: {.check}
+- [ ] Prøv å endre argumentene til `background` til du finner en farge du liker.
+
+# Steg 2: Hente inn bilde {.activity}
+
+I Processing kan et bilde bli lagret i en variabel av typen `PImage`, og hentes inn ved å bruke metoden `loadImage()`. Denne metoden tar navnet på bildet som et argument, og brukes gjerne inne i `setup`.
+
+![](bilde.png)
+
+Det neste steget nå er å tegne bilde på skjermen, det gjør vi ved hjelp av metoden `image()`. `image` er en metode som tar fem argumenter: `image(img, x, y, b, h)`
++ img er bildet vi vil tegne, feks `bilde1`
```

Comment:
bytte ut _+_ med _- [ ]_ for alle checkpoints her. evt bytt ut avsnittet med en seksjon med *Forklaring*, slik som du har ovenfor.